tests/Rector/v12/v0/ReplaceTSFECheckEnableFieldsRector/Fixture/fixture.php.inc

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Ssch\TYPO3Rector\Tests\Rector\v12\v0\ReplaceTSFECheckEnableFieldsRector\Fixture;
+
+use TYPO3\CMS\Frontend\Controller\TypoScriptFrontendController;
+
+$row = [];
+
+$foo = $GLOBALS['TSFE']->checkEnableFields($row);
+
+/** @var TypoScriptFrontendController $typoscriptFrontendController */
+$typoscriptFrontendController = $GLOBALS['TSFE'];
+$bar = $typoscriptFrontendController->checkEnableFields($row);
+$baz = $typoscriptFrontendController->checkEnableFields($row, true);
```

Comment:
There are more test cases missing, for example `checkPagerecordForIncludeSection` also got migrated

Reply:
Good catch, did not check, only the docs..